src/Microsoft.VisualStudio.Editor.Razor/RazorDirectiveCompletionProvider.cs

```diff
@@ -207,4 +199,40 @@ protected virtual bool TryGetRazorSnapshotPoint(CompletionContext context, out S
             return false;
         }
     }
+
+    // These types are only for this class to provide indirection for assembly loads.
+    internal abstract class CompletionProviderDependencies
```

Comment:
@rynowak if you know of a better way to do this I'm all ears 😉

Reply:
No, this seems about right.